src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
                 absMax = numbers[i];
             }
         }
-        return absMax;
+        return Math.abs(absMax);
```

Comment:
**Optimization Issue**: The updated logic in getMaxValue incorrectly identifies the absolute maximum value. The modification from checking the absolute value of each number to directly comparing the numbers without their absolute values can result in incorrect behavior for negative numbers. <br> **Fix**: Restore the use of Math.abs for both the current number and the current maximum when comparing values inside the loop. This ensures that the function correctly identifies the absolute maximum value, regardless of the sign of the input numbers. <br> **Code Suggestion**: 
 ```
 -        if (numbers[i] > absMax) {
 +        if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 -        return Math.abs(absMax);
 +        return absMax;
 ```